.github/workflows/ffigen.yml

```diff
@@ -27,25 +30,25 @@ jobs:
         working-directory: pkgs/ffigen/
     strategy:
       fail-fast: false
-      matrix:
-        sdk: [dev]
-#        sdk: [3.3.0]
     steps:
       - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11
       - uses: dart-lang/setup-dart@fedb1266e91cf51be2fdb382869461a434b920a3
         with:
-          sdk: ${{ matrix.sdk }}
+          sdk: 3.3.0
```

Comment:
I don't believe this has the deeply immutable.
So when starting to use that, this should probably be flipped to dev again.